### ISSUE TITLE
Orbital not capable of what Billboard can do (which is not a solver)

### ISSUE THREAD

**Issue by Alexees** [microsoft/MixedRealityToolkit-Unity#5142]
## Describe the bug

Orbital  does not have any option to follow one object while looking at another on the y-axis only. The Orbital script has all sorts SolverOrientationType modes instead, which completely define the rotation.
Orbital does not have a TransformTarget field even though it can for example look at the camera, with is an option of SolverOrientationType  and is hard coded.

## Expected behavior

Orbital should use the same PivotAxis enum like Billboard does.
In addition, it should be able to look at custom targets, like Billboard does
Basically, it should integrate the capability of being billboard, which it nearly is.

Or maybe creating a Billboard Solver and just ditch Orbital

Also, there are quite a lot of enums and a couple of them are pretty much either a portion of another or a combination of them:

OrientationType
PivotAxis
RadialViewReferenceDirection
RotationConstraintType
SolverOrientationType

And why doesn't the Pivot Axis enum make use of the FlagsAttribute?

**Comment by julenka** [comment]
Closing in favor of larger refactor #5713
